core/thread/inc/ROOT/TReentrantRWLock.hxx

```diff
@@ -123,6 +124,67 @@ struct RecurseCounts {
    size_t GetLocalReadersCount(local_t &local) { return fReadersCount[local]; }
 
 
+};
+
+struct RecurseCountsLocalSpin {
+   using ReaderColl_t = std::unordered_map<std::thread::id, int>;
+   size_t fWriteRecurse; ///<! Number of re-entry in the lock by the same thread.
+
+   std::thread::id fWriterThread; ///<! Holder of the write lock
+   ReaderColl_t fReadersCount;    ///<! Set of reader thread ids
+
+   std::atomic_flag fSpinLock;    ///<! MSVC doesn't support = ATOMIC_FLAG_INIT;
+
+   using local_t = std::thread::id;
+
+   RecurseCountsLocalSpin() {
+      // MSVC doesn't support fSpinLock=ATOMIC_FLAG_INIT; in the class definition
```

Comment:
This comment is redundant with the one 5 lines above.

Reply:
Sorta.  Again this is copy/paste of similar use and is 'useful' to keep in case somebody chose to outline the function later. (and will not think about re-adding the now not so redundant comment)